tests/vmi_hook_sidecar_test.go

```diff
@@ -213,8 +220,28 @@ var _ = Describe("[sig-compute]HookSidecars", decorators.SigCompute, func() {
 			})
 		})
 
+		Context("with ConfigMap in sidecar hook annotation", func() {
+			BeforeEach(func() {
+				vmi.ObjectMeta.Annotations = RenderSidecarWithConfigMap(hooksv1alpha2.Version)
+			})
+
+			It("should update domain XML with SM BIOS properties", func() {
+				_, err = virtClient.CoreV1().ConfigMaps(vmi.Namespace).Create(context.TODO(), RenderConfigMap(), metav1.CreateOptions{})
+				Expect(err).ToNot(HaveOccurred())
+				vmi, err = virtClient.VirtualMachineInstance(testsuite.GetTestNamespace(nil)).Create(context.Background(), vmi)
+				Expect(err).ToNot(HaveOccurred())
+				libwait.WaitForSuccessfulVMIStart(vmi)
```

Comment:
nit: this can be simplified:
```suggestion
				vmi = tests.RunVMIAndExpectLaunch(vmi, <timeout>)
```